config/generic-duet3-1lc.cfg

```diff
@@ -0,0 +1,80 @@
+# This file contains common pin mappings for the Duet3 1LC. To use
+# this config, the firmware should be compiled for the SAMC21G18 with:
+# Bootloader offset of "No Bootloader"
+# Clock Reference of "25 Mhz crystal"
+# Communication interface of "CAN bus (on PA25/PA24)"
+
+# To flash the board use a debugger, or use a raspberry pi and follow
+# the instructions at klipper/scripts/flash-atsamc21.sh. You may
+# supply power to the 1LC by connecting the 5v rail on the Pi to the
```

Comment:
I would suggest using the 3.3V supply out of an abundance of caution - Pi GPIO is fragile.

Reply:
I haven’t tried this, honestly not sure if it will work…

Reply:
It works - the CAN transceiver won't operate at 3.3V but the MCU runs just fine.